packages/flame/test/components/position_component_test.dart

```diff
@@ -404,6 +404,53 @@ void main() {
       expect(component.positionOf(Vector2(10, 0)), Vector2(44, 22));
     });
 
+    test('double center flips', () {
+      final startPosition = Vector2(50, 20);
+      final component = PositionComponent()
+        ..size = Vector2(10, 20)
+        ..angle = 2
+        ..scale = Vector2(2.0, 3.0)
+        ..position = startPosition;
+      final centerPosition = component.center;
+
+      component.flipVerticallyAroundCenter();
+      // Same position after one vertical flip.
+      expectVector2(component.center, centerPosition);
```

Comment:
let's add `epsilon=1e-15`, to check that they are actually equal up to a floating-point error